lib/serviceClient.ts

```diff
@@ -365,6 +366,7 @@ function createDefaultRequestPolicyFactories(credentials: ServiceClientCredentia
   if (!options.noRetryPolicy) {
     factories.push(exponentialRetryPolicy());
     factories.push(systemErrorRetryPolicy());
+    factories.push(throttlingRetryPolicy());
```

Comment:
I'd recommend making the throttling retry policy come after the deserialization policy. I know that you'll have to check !options.noRetryPolicy twice, but it makes sense that we shouldn't try to deserialize a 429 response.